test/unit/ServerConfigParams.test.ts

```diff
@@ -1,3 +1,125 @@
+import {
+  verifyServerConfiguration,
+  AppConfig,
+  ContractsConfig,
+  ServerConfigParams,
+  ERROR_DISABLE_SPONSOR_TX_NOT_CONFIGURED,
+  ERROR_GAS_FEE_PERCENTAGE_NEGATIVE,
+  ERROR_TRANSFER_FEE_PERCENTAGE_NEGATIVE,
+  ERROR_FIXED_USD_FEE_NEGATIVE,
+} from '../../src/ServerConfigParams';
+import { ethers } from 'ethers';
+import { expect, use } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+
+use(chaiAsPromised);
+
+describe('ServerConfigParams tests', function () {
+  describe('Function verifyServerConfiguration()', function () {
+    let contracts: ContractsConfig;
+    let app: AppConfig;
+    let randomAddress: string;
+
+    before(function () {
+      randomAddress = ethers.Wallet.createRandom().address;
+    });
+
+    beforeEach(function () {
+      app = {
+        url: 'https://fake.rsk',
+        port: 8080,
+        workdir: '/some/dir',
+      } as AppConfig;
+
+      contracts = {
+        relayHubAddress: randomAddress,
+      } as ContractsConfig;
+    });
+
+    it('Should fail if disableSponsoredTx is not properly configured', function () {
+      expect(() =>
+        verifyServerConfiguration({ app, contracts } as ServerConfigParams)
+      ).to.throw(ERROR_DISABLE_SPONSOR_TX_NOT_CONFIGURED);
+    });
```

Comment:
this shouldn't be possible beause we have a default.json5 config file.

Reply:
Agreed on put this on a separated task.